Tests/RevenueCatUITests/BaseSnapshotTest.swift

```diff
@@ -55,7 +55,7 @@ class BaseSnapshotTest: TestCase {
                            customerInfo: TestData.customerInfo,
                            mode: mode,
                            fonts: fonts,
-                           introEligibility: eligibleChecker,
+                           introEligibility: introEligibility,
```

Comment:
This needs to be the parameter. This silently worked because of implicit `self` 😢